core/ui/src/views/settings/SettingsSmartHost.vue

```diff
@@ -53,7 +53,54 @@
                 {{ $t("common.enabled") }}
               </template>
             </cv-toggle>
-            <template v-if="enabled === true">
+            <template v-if="enabled">
+              <p class="radio-menu-label">
```

Comment:
You can use `<label class="bx--label">` instead of this `p`. Then let's remove `radio-menu-label` CSS class